src/db/__init__.py

```diff
@@ -43,6 +43,26 @@ def get_next_survey_question(user_id, survey_id):
     return question, answers
 
 
+def get_survey_answer_id_by_alias(alias):
+    survey_answer_id = _fetchone(
+        f"SELECT survey_answer_id FROM survey_answer WHERE alias='{alias}'"
+    )
+    print("xxx " + str(survey_answer_id))
+    return _get_first_or_none(survey_answer_id)
+
+
+def check_if_survey_exists(survey_id):
```

Comment:
I think the `if` in the name is redundant. `check_survey_exists` reads a bit easier IMHO.

Reply:
Ok